core/lib/dal/src/blocks_dal.rs

```diff
@@ -2394,6 +2396,29 @@ impl BlocksDal<'_, '_> {
         .flatten())
     }
 
+    pub async fn get_sealed_at(
+        &mut self,
+        l1_batch_number: L1BatchNumber,
+    ) -> DalResult<Option<NaiveDateTime>> {
+        Ok(sqlx::query!(
+            r#"
+            SELECT
+                sealed_at
+            FROM
+                l1_batches
+            WHERE
+                is_sealed
```

Comment:
Isn't this condition redundant? If `!is_sealed`, `sealed_at` should be null. OTOH, it could still be null for old sealed batches; I think this is accounted for, right?

Reply:
removed